RentallCarsAPI/Models/Car.cs

```diff
@@ -10,5 +10,6 @@ public class Car
         public Brand Brand { get; set; }
         public string Color { get; set; }
         public Transmition Transmition { get; set; }
+        public bool isFree { get; set; }
```

Comment:
Para continuar con las mismas convenciones
```suggestion
        public bool IsFree { get; set; }
```

Reply:
done